### PR TITLE
Food List

### PR DESCRIPTION
**Pull request by ericmahare**
In this milestone I did the following: -

- [x] Added a functionality display a list of food added by the logged-in user 
- [x] Created a form that allows users to add new food

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,16 @@ rails s
...
+👤 **Chia Carlyle**
+- GitHub: [![GitHub Badge](https://img.shields.io/badge/-carlylechia-black?logo=GitHub&logoColor=18171&style=plastic)](https://github.com/ericmahare)
+
+- LinkedIn: [![LinkedIn Badge](https://img.shields.io/badge/-chiacarlyle-black?logo=LinkedIn&logoColor=0A66C2&style=plastic)](https://www.linkedin.com/in/eric-mahare-358944183/)
+
+- Gmail: [![Gmail Badge](https://img.shields.io/badge/-chiacarlyle-black?logo=Gmail&logoColor=EA4335&style=plastic)](mailto:erikmahare@gmail.com)
+
+- Twitter: [![Twitter Badge](https://img.shields.io/badge/-chiacarlyle-black?logo=Twitter&logoColor=1DA1F2&style=plastic)](https://twitter.com/erikmahare)
+
+
```
Comment (carlylechia) [inline README.md]: Please, do well to update the readme file with your correct information.

File: app/assets/stylesheets/scaffold.css [hunk 2]
```diff
@@ -127,3 +127,168 @@ footer {
...
 }
+
+h2 {
+  text-align: center;
+  font-size: 18px;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  color: white;
+  padding: 30px 0;
+}
```
Comment (carlylechia) [inline app/assets/stylesheets/scaffold.css]: Great styling on the food section. However, this h2 styling here over-writes the previous styles for that HTML tag and it causes some styling issues in other sections, like the recipes and public recipes. What if you found a more specific way to reference that h2 tag, maybe give it a class?
Furthermore, I suggest that you kindly transfer the styling specific to the food section into a separate file, maybe foods.css, for better modularization.

**Review by carlylechia — APPROVED** [review]
## Hi @ericmahare 
Your project is complete! Before you proceed to merge it though, please kindly look at the comments attached below, as they might make our code even better.  :shipit:
Congratulations! 🎉 👏 👏 💯

image



**To highlight**🍾

- All linters are passing ✔️
- Good commit history ✔️
- Good PR description ✔️

## Optional suggestions

 

- [ ] Kindly Check the comments under the review._

*Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.*
**Cheers and Happy coding**!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.[^1]

[^1]:  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/assets/stylesheets/scaffold.css (hunk 2 above)
Comment (ericmahare) [reply app/assets/stylesheets/scaffold.css]: Alright  I will work on that

**Pull request merged**